### ISSUE TITLE
Add-IntuneWin32AppAssignmentGroup Error on setting Restart

### ISSUE THREAD

**Issue by acondegni** [MSEndpointMgr/IntuneWin32App#58]
It is not possible to use Add-IntuneWin32AppAssignmentGroup to mimic the same thing that is allowed via the console.   

When trying to use:  
Add-IntuneWin32AppAssignmentGroup -Include -ID SOMEID -GroupID SOMEGROUPID -Intent available -Notification showAll 
 DeliveryOptimizationPriority foreground -EnableRestartGracePeriod $true -RestartGracePeriod 180 -RestartCountDownDisplay 15

The following warning is displayed:
WARNING: Win32 app was not configured for restart settings, ensure restart behavior is configured with 'Based on return code'

It is forcing use of "Device Restart Behavior" setting of "Determine Behavior based on return codes".   But when creating the assignment with the console, I am successfully able to create the assignment with a "Device Restart Behavior" = "No Specific Action".

I was able manually update the code and remove the dependency on return codes.   Doing that allowed a successful creation of the assignment.

**Comment by NickolajA** [comment]
Yeah this sounds like a mistake on my end. Could you share the code changes please?
